CommonMC/src/CosmicTimeOffset_module.cc

```diff
@@ -31,6 +32,8 @@ namespace mu2e {
       using Name=fhicl::Name;
       using Comment=fhicl::Comment;
 
+      fhicl::Atom<std::string> cosmicModuleLabel{Name("cosmicModuleLabel"), Comment("Name of CORSIKA module label"), "generate"};
```

Comment:
Isn't this more general than CORSIKA?  Also, shouldn't this be an optionalAtom, as we don't need it except for Extracted.  If you make it optional then you don't need the explicit boolean parameter.

Reply:
Yes, sorry, the comment is wrong and yes, it should be Optional